src/client/App.js

```diff
@@ -1,12 +1,69 @@
 import React, { Component } from 'react';
 
+const MostUsedWords = React.createClass({
+		'getInitialState': function () {
+			return {
+				showAllWords: true,
+				'words': [
+					{ 'word': 'three', 'count': 3 },
+					{ 'word': 'two', 'count': 2 },
+					{ 'word': 'one', 'count': 1 },
+				]
+			};
+		},
+		'componentDidMount': () => {
+			$(".ui.checkbox").checkbox();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2176021679/issues/source?bid=3148782&fileBranchId=3163407#l15)